modules/@apostrophecms/oembed-field/ui/apos/components/AposInputOembed.vue

```diff
@@ -104,7 +112,7 @@ export default {
       this.validateAndEmit();
     },
     async loadOembed () {
-      this.field.readOnly = true;
+      this.tempReadOnly = true;
```

Comment:
UX improvement: do not disable the input otherwise we lose the focus when typing (ie: when disabled).
To do that, use another variable and do not mutate `this.field.readOnly`.

On the other hand, `this.field.readOnly` will disable the input if set to `true` and that's okay, since we are not to edit at all